contracts/vault/AssetVault.sol

```diff
@@ -118,4 +130,18 @@ contract AssetVault is IAssetVault, OwnableERC721, Initializable, ERC1155Holder,
         payable(to).sendValue(balance);
         emit WithdrawETH(msg.sender, to, balance);
     }
+
+    /**
+     * @inheritdoc IAssetVault
+     */
+    function call(address to, bytes calldata data) external override onlyWithdrawDisabled nonReentrant {
```

Comment:
What does the flow look like to actually make his call? Is `LoanCore` the `owner()` here, or does the owner have to implement `ICallDelegator` somehow? Not quite sure because I was imaging the owner was an end user (possibly an EOA)

Reply:
If the owner is an EOA, they can just call stuff on the vault at will. If the owner is a contract ie when in use as collateral, then the contract has to implement this fn to grant call privs